tests/entry_tests.rs

```diff
@@ -16,7 +16,7 @@ mod tests {
             assert_eq!(e.get_url(), Some("http://keepass.info/"));
             assert_eq!(e.get("custom attribute"), Some("data for custom attribute"));
             assert_eq!(e.get("URL"), Some("http://keepass.info/"));
-            assert_eq!(e.expires, false);
+            assert_eq!(e.times.expires, false);
```

Comment:
@sseemayer this is a breaking change. Ideally we would mark the commit message with `BREAKING CHANGE`, but now that I read the [semantic-release docs](https://github.com/semantic-release/semantic-release#commit-message-format),it looks like this would bump the major version. This is not what we want, at least not until we reach v1.
I think we can skip the `BREAKING CHANGE` for now, but we should make it clear in the commit message that
the field was moved.

Reply:
If you manually release `0.6.0`, then I think we are good. Otherwise semantic-release would pick up all the commit since the last release and create a release with that, and we would get a `1.0.0` because of the breaking change.